engines/draci/draci.cpp

```diff
@@ -47,6 +49,56 @@
 
 namespace Draci {
 
+// Used by all languages but Polish
+static const int kamenickyEncodingTable[] = {
```

Comment:
maybe wrap this in `#ifdef USE_TTS` and maybe use `uint16` to reduce the memory needed by the engine in some cases.